DESCRIPTION

```diff
@@ -36,11 +36,10 @@ Suggests:
     knitr,
     rmarkdown,
     covr,
-    AmesHousing,
     spelling,
     xml2,
     modeldata
 Roxygen: list(markdown = TRUE)
-RoxygenNote: 7.1.0
+RoxygenNote: 7.1.1.9000
```

Comment:
I couldn't find version `7.1.1.900` of Roxyen, only `7.1.1`.  Can we still use `7.1.1.900` anyways?

Reply:
It's the development version. This is ok since it is used to generate the man files so it won't be an issue for CRAN of GitHub actions.